Lib/unittest/mock.py

```diff
@@ -1600,17 +1627,20 @@ def __enter__(self):
 
         new_attr = new
 
-        self.temp_original = original
-        self.is_local = local
-        self._exit_stack = contextlib.ExitStack()
+        self._started_context = _PatchStartedContext(
+            exit_stack=contextlib.ExitStack(),
+            is_local=is_local,
+            target=self.getter(),
+            temp_original=original,
```

Comment:
This attribute can be called `original` since we are anyway wrapping it in a new object.